terraform/lambda/variables.tf

```diff
@@ -2,14 +2,19 @@ variable "environment" {
   type = string
 }
 
+variable "use_localstack" {
+  type = bool
+  default = false
+}
+
 variable "lambda_zip_file" {
   default     = "../../../../di-ipv-core-back/build/distributions/di-ipv-core-back.zip"
   description = "Location of the Lambda ZIP file"
   type        = string
 }
 
 locals {
-  default_tags = {
+  default_tags = var.use_localstack ? null : {
```

Comment:
just a question: does localstack not work with these tags set?

Reply:
It doesn't seem like it does from a bit of a google. And this is also the same thing that the auth team do for their localstack stuff. Without this basically the terraform apply command will just get stuck at various places and end up failing.